src/tray.ts

```diff
@@ -14,134 +14,116 @@ import { t } from '@/i18n';
 
 import type { MenuTemplate } from './menu';
 
-// Prevent tray being garbage collected
```

Comment:
Is there a reason this comment was removed?

Reply:
Added back. Forgot to add back after refactoring the codebase.